test/controllers/contactDetails/CharityNameControllerSpec.scala

```diff
@@ -58,11 +66,11 @@ class CharityNameControllerSpec extends SpecBase with BeforeAndAfterEach {
   private val formProvider: CharityNameFormProvider = injector.instanceOf[CharityNameFormProvider]
   private val form: Form[CharityName]               = formProvider()
 
-  private val controller: CharityNameController = inject[CharityNameController]
-
   "CharityNameController" must {
 
-    "redirect to EmailOrPost page when acknowledgement reference is present" in {
+    "redirect to EmailOrPost page when acknowledgement reference is present and the feature is turned on" in new Setup(
```

Comment:
Do you mean turned off? I would think if it is turned on there would be no email or post feature.